test/rest_api/README.md

```diff
@@ -0,0 +1,35 @@
+Tests for the Scylla REST API.
+
+Tests use the requests library and the pytest frameworks
+(both are available from Linux distributions, or with "pip install").
+
+To run all tests using test.py, just run `./test.py api/run`.
```

Comment:
is it now rest_api/run?

Also I would just run "test/rest_api/run". No need to ask test.py to do it ;-) Although you're right, you can.

Reply:
Also this line and the "more conveniently, ..." paragraph below talk about basically the same thing.

Reply:
I never remember the pytest details as I rarely use it, hence the ./test.py instructions for dummies like me.

Reply:
api -> rest_api fixed in the next version

Reply:
also in https://github.com/scylladb/scylla/issues/9754 that will introduce it

Reply:
Done in #9755